app/scripts/metamask-controller.js

```diff
@@ -545,7 +545,7 @@ export default class MetamaskController extends EventEmitter {
         ),
       onNetworkDidChange: networkControllerMessenger.subscribe.bind(
         networkControllerMessenger,
-        'NetworkController:stateChange',
```

Comment:
@Gudahtt There is a problem with this change. It requires https://github.com/MetaMask/core/pull/3598, which has not been released yet.

Will leaving this as `'NetworkController:stateChange'` be buggy? Or does it just mean that the listener will be called more than it needs to be?

Reply:
Actually, I addressed this here c65def9baf

Reply:
Good catch. We would have released the network controller alongside this had we noticed that.